<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeTool</class>
 <widget class="QWidget" name="VolumeTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="volumeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#volumeWidget{
	background-color:#ffffff;
	border-radius:5px;
}</string>
   </property>
   <widget class="QPushButton" name="silenceBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#silenceBtn { border:none; }
#silenceBtn:hover{ background-color:#F0F0F0; }</string>
    </property>
    <property name="text">
     <string>静音</string>
    </property>
   </widget>
   <widget class="QLabel" name="volumeRatio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="inSlider">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>25</y>
      <width>4</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#inSlider { background-color:#ECECEC; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="outSlider">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>25</y>
      <width>4</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#outSlider { background-color:#1ECC94; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="sliderBtn">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>20</y>
      <width>14</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#sliderBtn
{
	background-color:#1ECC94; 
	border-radius:7px;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
